server/routes/api/place.py

```diff
@@ -113,6 +114,56 @@ def child(dcid):
     return json.dumps(child_places)
 
 
+# Defines the map from higher geos to their respective subgeos.
+LEVEL_MAP = {
```

Comment:
If this is the same as in another file, it would be better to define it once and import it.